stacks/carpark-stack.js

```diff
@@ -28,6 +28,7 @@ export function CarparkStack({ stack, app }) {
   const { eventBus } = use(BusStack)
 
   const carparkBucket = new Bucket(stack, 'car-store', {
+    cors: true,
```

Comment:
see: https://docs.sst.dev/constructs/Bucket#cors

Reply:
`cors: true` means allow everything from everywhere (only in CORS land tho, access still requires auth)
https://github.com/serverless-stack/sst/blob/9854bbf09e82bb05dae24b76162c08a44540f67e/packages/resources/src/Bucket.ts#L643-L654